tests/run-pass/aux_test.rs

```diff
@@ -1,4 +1,5 @@
 // aux-build:dep.rs
+// This ignores the test against rustc, but runs it against miri:
```

Comment:
Well, the test fails when it is tested against rustc. I figured `ignore-cross-compile` was just a hack to make it even possible to test things against miri but not rustc.